### ISSUE TITLE
Better checkout workflow.

### ISSUE THREAD

**Issue by ideadude** [strangerstudios/pmpro-gift-levels#17]
The following workflow has been suggested. Should we support this? How could we support this?

When checking out for a desired level, add an option at checkout "Is this a gift?" If set to "Yes" then add additional fields to collect the gift recipient's name and email address. Maybe a gift message. Then after completing checkout, send an email to the gift recipient with the link to sign up and login.

**Comment by kimwhite** [comment]
Good idea. I just worked with customer to add gist to do some of this.

**Comment by mircobabini** [comment]
> Good idea. I just worked with customer to add gist to do some of this.

Sounds interesting. Can you please share the gist(s) to give it a try?

**Comment by dparker1005** [comment]
Suggestion to send email to gift receiver is now being represented by #35
